mass_mailing_partner/models/mailing_contact.py

```diff
@@ -106,7 +106,7 @@ def _set_partner(self):
         email = self.email.strip()
         partner = m_partner.search([("email", "=ilike", email)], limit=1)
```

Comment:
suggestion: you can improve performance if you move the new check up here. You'll save some queries:

```suggestion
        if email == self.partner_id.email:
            return
        partner = m_partner.search([("email", "=ilike", email)], limit=1)
```

If so, remember to leave line 109 as it was.

Reply:
Sure, that's better. Done :slightly_smiling_face: